src/styles/theme.ts

```diff
@@ -53,8 +53,6 @@ const components = {
     }),
   },
   Button: { ...buttonStyles },
-  Drawer: { ...priceBarDrawer },
-  DrawerButton,
 };
```

Comment:
I was wondering why you need to delete those code?

Reply:
Update later. Might be caused by dealing with conflict wrongly.

Reply:
already update and fixed it.